sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/PartitionPump.cs

```diff
@@ -19,17 +19,23 @@ namespace Azure.Messaging.EventHubs.Processor
     internal class PartitionPump
     {
         /// <summary>The <see cref="EventHubRetryPolicy" /> used to verify whether an exception is retriable or not.</summary>
-        private static readonly BasicRetryPolicy s_retryPolicy = new BasicRetryPolicy(new RetryOptions());
+        private static readonly BasicRetryPolicy RetryPolicy = new BasicRetryPolicy(new RetryOptions());
```

Comment:
This should come in as part of the Event Processor Options;  there should be a mandatory `RetryOptions` member that you can call `ToRetryPolicy` on after my changes go in.

Reply:
Doesn't need to block this; we can put it in after mine hits, even if it doesn't make preview 5.

Reply:
Just took a note.